app/uk/gov/hmrc/excisemovementcontrolsystemapi/services/MessageService.scala

```diff
@@ -71,6 +74,10 @@ class MessageService @Inject() (
     lockService
       .withLock {
         val now = dateTimeService.timestamp()
+        // Temporary migration code to update movement's lastUpdated
+        if (shouldMigrateLastUpdated(lastRetrieved)) {
+          movementRepository.migrateLastUpdated(ern)
```

Comment:
This isn't mapped on anything, if it fails we'll never know

Reply:
added a recover with a logged warning incase it fails